src/ontology/uberon.Makefile

```diff
@@ -1238,7 +1254,7 @@ $(BRIDGEDIR)/%.owl: $(BRIDGEDIR)/%.obo
 make-bridge-ontologies-from-xrefs.pl:
 	cp $(SCRIPTSDIR)/make-bridge-ontologies-from-xrefs.pl $@
 
-$(TMPDIR)/bridges: $(TMPDIR)/seed.obo $(TMPDIR)/cl-with-xrefs.obo $(TMPDIR)/cl-zfa-xrefs.obo $(BRIDGEDIR)/uberon-bridge-to-nifstd.owl make-bridge-ontologies-from-xrefs.pl $(REPORTDIR)/life-cycle-xrefs.txt
+$(TMPDIR)/bridges: $(TMPDIR)/seed.obo $(TMPDIR)/cl-with-xrefs.obo $(TMPDIR)/cl-zfa-xrefs.obo $(BRIDGEDIR)/uberon-bridge-to-nifstd.owl make-bridge-ontologies-from-xrefs.pl $(REPORTDIR)/life-cycle-xrefs.txt $(BRIDGEDIR)/uberon-bridge-to-mba.owl $(BRIDGEDIR)/uberon-bridge-to-dmba.owl $(BRIDGEDIR)/uberon-bridge-to-mba.obo $(BRIDGEDIR)/uberon-bridge-to-dmba.obo
```

Comment:
Good, this should be enough to have the custom MBA/DMBA bridges fetched at the same time as we build the other bridges.

Minor nitpick: Instead of listing the MBA/DMBA bridges here, I would rather have another Make variable (say, `CUSTOM_BRIDGES` or something similar) where we could list all the bridges that are generated by custom rules, and then we reference that variable in the dependencies of `$(TMPDIR)/bridges`.

Reply:
things I don't think about and @hkir-dev ends up having to clean all my ugly code lol >.<